### PR TITLE
Made some grammar fixes and added version number adding suggestion to the issue template.

### PR DESCRIPTION
**Pull request by pahakalle**

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
...
  
  ### Issue / bug
- - Explain the issue / bug as easily understandably as possible.
+ - Explain the issue / bug as understandably as possible.
+ - If possible add minecraft, forge, java and modpack version numbers for debugging (mod authors might need these).
  - If the issue/bug is recurring explain how it can be recreated (helps darkosto!).
- - Add a error/crash log to [Pastebin](https://pastebin.com/), and link it here, if possible(if you disslike pastebin any other similiar site is fine too)
- - Add a video/gif of the issue/bug if possible (makes it easier for darkosto to understand you).
\ No newline at end of file
+ - If possible add error/crash log to [Pastebin](https://pastebin.com/) and link it here (if you disslike pastebin any other similiar site is fine too).
```
Comment (tomudding) [inline ISSUE_TEMPLATE.md]: _Sorry for this second review request but GitHub sometimes **dislikes** what I do._
`disslike` should be `dislike`

Comment (pahakalle) [reply ISSUE_TEMPLATE.md]: Oh, i swear i fixed that

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -4,5 +4,5 @@
  - Explain the issue / bug as understandably as possible.
  - If possible add minecraft, forge, java and modpack version numbers for debugging (mod authors might need these).
  - If the issue/bug is recurring explain how it can be recreated (helps darkosto!).
- - If possible add error/crash log to [Pastebin](https://pastebin.com/) and link it here (if you disslike pastebin any other similiar site is fine too).
- - If possible add a video/gif of the issue/bug (makes it easier for darkosto to understand you).
\ No newline at end of file
+ - If possible add error/crash log to [Pastebin](https://pastebin.com/) and link it here (if you dislike pastebin any other similiar site is fine too).
```
Comment (tomudding) [inline ISSUE_TEMPLATE.md]: And apparently `similar` isn't written as `similiar` ;)

Comment (pahakalle) [reply ISSUE_TEMPLATE.md]: Doing good grammar is hard when your tired and a bit drunk 😂

**Pull request merged**